dts/bindings/pwm/adi,max6639-pwm.yaml

```
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2025 Tenstorrent AI ULC
+#
+# SPDX - License - Identifier : Apache - 2.0
+#
+
+description: |
+    ADI MAX6639 PWM Fan-Speed Controller, PWM device
+
+compatible: "adi,max6639-pwm"
```

Keep the Maxim branding, which is also consistent with what is upstream.
```suggestion
compatible: "maxim,max6639-pwm"
```